### PR TITLE
behat step to check displayname in WebUI

### PR DESCRIPTION
**Pull request by individual-it**
## Description
one simple step to check if the shown displayname in the WebUI (top right corner) is as expected

## Motivation and Context
when changing the displayname, we want to be able to check if its displayed correctly

## How Has This Been Tested?
created UI tests (in user_ldap) that use that new step

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/ui/features/bootstrap/WebUIUserContext.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+use Behat\Behat\Context\Context;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Page\OwncloudPage;
+
+require_once 'bootstrap.php';
+
+/**
+ * WebUIUser context.
+ * Context for steps associated with the User logedin into the WebUI
```
Comment (phil-davis) [inline tests/ui/features/bootstrap/WebUIUserContext.php]: s/logedin into/logged in to/

**Review by phil-davis — COMMENTED** [review]
1 comment typo

File: tests/ui/features/bootstrap/WebUIUserContext.php (hunk 1 above)
Comment (individual-it) [reply tests/ui/features/bootstrap/WebUIUserContext.php]: fixed

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport stable10 #30662

**Pull request merged**